Formula/smenu.rb

```diff
@@ -0,0 +1,18 @@
+class Smenu < Formula
+  desc "Command-line tool that turns its input into interactive selection menus"
+  homepage "https://github.com/p-gen/smenu"
+  url "https://github.com/p-gen/smenu/archive/v0.9.9.tar.gz"
```

Comment:
Is this project considered stable? Per our [formulae requirements](https://github.com/Homebrew/brew/blob/master/docs/Acceptable-Formulae.md#stable-versions), we don't accept beta or prerelease versions in core.